book/version_control.md

```diff
@@ -34,13 +34,27 @@ When used effectively, version control helps us to identify which changes have n
 
 Git is invaluable when recording and combining changes from multiple developers, as merging allows us to resolve conflicts between overlapping changes. Using a remote Git repository maintains a single source of truth, even when multiple individuals are working on a project. Additionally, version control platforms, like GitHub and GitLab, can make it easier to track and review ongoing work. This avoids duplication of effort and keeps review embedded in the development workflow.
 
-## Include or exclude
+## Include code and documentation in version control
 
-Ideally, you should include any code that is required to run your analysis. In a public repository, you may need to omit confidential or sensitive aspects of the project.
+Ideally, you should include all code and documentation that is required to run your analysis.
+
+This may include:
+
+* Project documentation including readme and contribution guidance
+* Packaged functions and code documentation
+* Unit tests
+* Dummy data and configuration files
+* User documentation such as desk notes and installation guidance
+
+Whilst inclusion of [configuration](configuration.md) files is needed to help users understand the parameters to be set, it is best to include example files. Since configuration files often contain sensitive data or credentials, which should **not** be committed to version control, example files should contain clear dummy values that are indicative of the expected values.
```

Comment:
Given the new heading I think we should update this to be a clear steer that you should include example configuration files. We want to give similar explanation for the other things listed here, so we might drop the explanation for configs? If we keep the explanation, it might need updating to say that configs are usually environment specific so aren't useful for other people wanting to reproduce your analysis

Then we could adjust the exclude list below to say that "real" config files should not be included. We could remove the part about environment-specific. Hopefully together these changes would make the guidance more simple to follow